testing/radar_control_input_sim/radar_control_input_sim.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/python
+import sys
+import time
+import threading
+import os
+
+
+BOREALISPATH = os.environ['BOREALISPATH']
+sys.path.append(BOREALISPATH)
+sys.path.append(os.environ["BOREALISPATH"] + '/build/release/utils/protobuf')
+
+from utils.zmq_borealis_helpers import socket_operations as so
+from utils.experiment_options.experimentoptions import ExperimentOptions
+
+from driverpacket_pb2 import DriverPacket
+from sigprocpacket_pb2 import SigProcPacket
+from rxsamplesmetadata_pb2 import RxSamplesMetadata
+from processeddata_pb2 import ProcessedData
+
+
+options = ExperimentOptions()
+
+ids = [options.dsp_to_radctrl_identity, options.driver_to_radctrl_identity, 
+        options.driver_to_dsp_identity, options.driver_to_brian_identity,
+        options.dsp_to_driver_identity, options.dspbegin_to_brian_identity, 
+        options.dspend_to_brian_identity]
+
+sockets_list = so.create_sockets(ids, options.router_address)
+
+dsp_to_radctrl = sockets_list[0]
+driver_to_radctrl = sockets_list[1]
+driver_to_dsp = sockets_list[2]
+driver_to_brian = sockets_list[3]
+dsp_to_driver = sockets_list[4]
+dsp_to_brian_begin = sockets_list[5]
+dsp_to_brian_end = sockets_list[6]
+
+def printing(err_msg):
+    print(err_msg)
+
+def dsp():
+    while True:
+        so.send_request(dsp_to_radctrl, options.radctrl_to_dsp_identity ,"Need metadata")
+        reply = so.recv_reply(dsp_to_radctrl, options.radctrl_to_dsp_identity,printing)
+
+        sigp = SigProcPacket()
+        sigp.ParseFromString(reply)
+        
+        so.send_request(dsp_to_driver, options.driver_to_dsp_identity, "Need data to process")
+        reply = so.recv_reply(dsp_to_driver, options.driver_to_dsp_identity, printing)
+
+        print(type(reply))
+        meta = RxSamplesMetadata()
+        meta.ParseFromString(reply) 
+
+        time.sleep(0.030)   
```

Comment:
it's Maaaagic

Reply:
Yeah, just a placeholder for real work i guess.